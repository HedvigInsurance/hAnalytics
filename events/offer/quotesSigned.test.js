const validateEvent = require("../../commons/validateEvent.js")

test('signed one SE_ACCIDENT quote', () => {
    const event = validateEvent(__filename, {
        quoteIds: ["123"]
    }, {
        quoteBundle: {
            quotes: [
                {
                    typeOfContract: "SE_ACCIDENT"
                }
            ]
        }
    })

    expect(event).toMatchSnapshot()
});

test('signed two quotes', () => {
    const event = validateEvent(__filename, {
        quoteIds: ["123", "12333"]
    }, {
        quoteBundle: {
            quotes: [
                {
                    typeOfContract: "SE_ACCIDENT"
                },
                {
                    typeOfContract: "SE_HOUSE"
                }
            ]
        }
    })

    expect(event).toMatchSnapshot()
});