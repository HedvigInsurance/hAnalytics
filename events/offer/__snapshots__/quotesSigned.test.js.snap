// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`signed one SE_ACCIDENT quote 1`] = `
Object {
  "name": "quotes_signed",
  "variables": Object {
    "initiated_from": "SELF_CHANGE",
    "quote_ids": Array [
      "123",
    ],
    "type_of_contracts": "SE_ACCIDENT",
  },
}
`;

exports[`signed two quotes 1`] = `
Object {
  "name": "quotes_signed",
  "variables": Object {
    "initiated_from": "APP",
    "quote_ids": Array [
      "123",
      "12333",
    ],
    "type_of_contracts": "SE_ACCIDENT, SE_HOUSE",
  },
}
`;
